netmanager/src/views/components/DataDisplay/DeviceManagement/ManagementMap.js

```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from "react";
```

Comment:
Just wondering....could we also review the organisational structure and have the deviceManagement under the views/pages directory?